scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -225,5 +232,47 @@ task ExtractTask {
      }
  }
 
+task GetLastModifiedDateOrUniqueValue {
+    # because this is being used to determine if the data has changed, never use call cache
+    meta {
+        volatile: true
+    }
+
+    input {
+        String data_project
+        String default_dataset
+        String dataset_table
+        File? service_account_json
```

Comment:
Does this mean that the file physically gets passed around to task runners (apologies if I'm not getting the architecture right)